perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -198,8 +215,23 @@ def PrepareVMEnvironment(self):
       if self.is_static:
         self.SnapshotPackages()
       self.SetupPackageManager()
+    self.SetFiles()
+    self.DoSysctls()
     self.BurnCpu()
 
+  def SetFiles(self):
+    """Apply --set_files to the VM."""
+
+    for pair in FLAGS.set_files:
+      path, value = pair.split('=')
+      self.RemoteCommand('echo "%s" | sudo tee %s' %
+                         (value, path))
+
+  def DoSysctls(self):
+    """Apply --sysctl to the VM."""
+
+    self.RemoteCommand('sudo sysctl -w %s' % (' '.join(FLAGS.sysctl),))
```

Comment:
Does this break with no sysctls specified? Even if it doesn't probably don't want to execute an unnecessary command